frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/PipelineRunTabDetails.tsx

```diff
@@ -51,9 +65,41 @@ const PipelineRunTabDetails: React.FC<PipelineRunTabDetailsProps> = ({
       ]
     : [];
 
+  const jobRefDescription =
+    data && data.description
+      ? [
+          {
+            key: 'Description',
+            value: data.description,
+          },
+        ]
+      : [];
+  const pipelineDescription = pipelineRunKF.description
+    ? [
+        {
+          key: 'Description',
+          value: pipelineRunKF.description,
+        },
+      ]
+    : [];
+  const renderRunName = {
+    key: 'Name',
+    value: (
+      <Split hasGutter>
+        <SplitItem>
+          <PipelineDetailsTitle run={pipelineRunKF} />{' '}
+        </SplitItem>
+        <SplitItem>
+          <PipelineRunTypeLabel resource={pipelineRunKF} />
+        </SplitItem>
+      </Split>
+    ),
+  };
```

Comment:
Did we not want to use the PipelineRun name here too? 
![image](https://github.com/opendatahub-io/odh-dashboard/assets/8126518/0bdb5410-d0c2-4805-baa2-680b83890b8e)

May be a bit redundant from the title -- but it is the current convention.

cc @yannnz

Reply:
@yannnz thanks for confirmation, I have added it back.

Reply:
Thanks! I am not sure which screenshot should I look at as the latest one, but the order of the info is Name, Description, then following Pipeline, Project. @uidoyen

Reply:
@yannnz here is the latest screenshot.

<img width="1424" alt="Screenshot 2023-08-31 at 10 16 14 PM" src="https://github.com/opendatahub-io/odh-dashboard/assets/2117670/8e61a479-56e8-4a50-ac89-0dea296c5386">

Reply:
Sorry I missed this message somehow. I just realized why don't we have Pipeline info in the Recurring run's detail? Is there a technical reason? @uidoyen

Reply:
Fixed -- it is in the Job not in the Runs created by Jobs like normal Runs.